src/schema/objects/seo.js

```diff
@@ -66,14 +66,20 @@ export const seo = {
       ]
     },
     {
-      title: 'Advanced',
+      title: 'Geavanceerd',
       name: 'advanced',
       type: 'object',
       options: {
         collapsible: true,
         collapsed: true
       },
       fields: [
+        {
+          title: 'Activeren',
+          name: 'enabled',
+          type: 'boolean',
+          initialValue: true,
+        },
```

Comment:
Willen we dat gebruikers dit aan en uit kunnen zetten?

Reply:
Het is een advanced feature; zal in de praktijk weinig voorkomen. Kan me wel voorstellen dat SEO op specifieke pagina's (404, afgeschermde content) helemaal niet van toepassing is en dus uitgezet kan worden.

Reply:
Voor afgeschermde content wil je denk ik juist de SEO plugin aan, zodat je noindex aan kunt zetten (zit dat er eigenlijk in?) Kun je niet `withSeo` weglaten als je geen seo wilt?

Reply:
@peeke Zeg jij dan eigenlijk dat we wat dat betreft beter in de default value `seo: { enabled: true }` kunnen zetten?

Reply:
Dit PR neigt inderdaad wel naar symptoom bestrijding tbh.

https://github.com/Kaliber/bol-com-adverteren/pull/261
^^ Andere benadering, maar denk ik wel een betere oplossing voor dit probleem.

Reply:
@coffeemedia Oeh nice! Kan ik me wel in vinden. 

@peeke Als we nou de `doc` bij nieuwe projecten al in de `groq` zetten, dan hoeven we ook niet derived te gebruiken voor elke doc. Laten we dat meenemen in de template.